plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -192,6 +192,12 @@ public String getFileName() {
   private JarArchiver jarArchiver;
 
   /**
+   * Allows enabling/disabling gcr credentials
+   */
+  @Parameter(defaultValue = "true", property = "dockerfile.googleregistry")
```

Comment:
@freesoft - my intention was to check for the creds if the property was missing, as that would maintain backward compatibility and it tested out that way.  Let me know if thats not appropriate.